### PR TITLE
Quest stuff

### PR DESCRIPTION
**Pull request by Prometheus0000**
Also, add recipes so you can craft the steam grinder in steam. Also fix the thaumium and void scoop recipes.

### CODE REVIEW

**Review by bookerthegeek — APPROVED** [review]

**Comment by Dream-Master** [comment]
if it is possible not mix up quest changes and script changes in one pr. Thanks

File: config/betterquesting/DefaultQuests.json [hunk 1]
```diff
@@ -8547,21 +8547,21 @@
...
-          "ignoreNBT:1": 0,
+          "ignoreNBT:1": 1,
           "index:3": 0,
           "consume:1": 0,
           "requiredItems:9": {
             "0:10": {
-              "id:8": "dreamcraft:tile.QuintupleCompressedCharcoal",
-              "Count:3": 1,
+              "id:8": "dreamcraft:tile.Charcoal",
+              "Count:3": 64,
```
Comment (leagris) [inline config/betterquesting/DefaultQuests.json]: Please change this to the gregtech version that is the recipe output from a compressor.
```json
              "id:8": "gregtech:gt.blockgem3:4",
              "Count:3": 64,
```

File: config/betterquesting/DefaultQuests.json [hunk 2]
```diff
@@ -319444,7 +319444,7 @@
       "home_image:8": "dreamcraft:textures/gui/gtnh_title.png",
       "party_enable:1": 1,
       "pack_version:3": 2105,
-      "editMode:1": 0,
+      "editMode:1": 1,
```
Comment (leagris) [inline config/betterquesting/DefaultQuests.json]: Edit mode shall not be in the commit.

**Review by leagris — CHANGES_REQUESTED** [review]

File: config/betterquesting/DefaultQuests.json (hunk 1 above)
Comment (Prometheus0000) [reply config/betterquesting/DefaultQuests.json]: Done

File: config/betterquesting/DefaultQuests.json (hunk 2 above)
Comment (Prometheus0000) [reply config/betterquesting/DefaultQuests.json]: Done

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**